graphql_compiler/compiler/expressions.py

```diff
@@ -227,6 +227,33 @@ def to_cypher(self):
         # Cypher has built-in support for variable expansion, so we'll just emit a variable
         # definition and rely on Cypher to insert the value.
         self.validate()
+
+        # The Neo4j client allows us to pass dates and datetimes directly as arguments. E.g. this
```

Comment:
Are you raising an error if somebody tries to pass in a date or datetime parameter? Also, I am against having this much documentation here on how to pass in datetime parameters for the neo4j library. I think that the overarching idea is that when you pass in parameters to neo4j you have to use the correct python type according to this link:
https://neo4j.com/docs/api/python-driver/  This documentation can be included elsewhere, but this doesn't seem to like the right place.

Reply:
What do you mean by raising an error? Including date(time) parameters along with the query is perfectly ok.

I'm open to cutting it down and simply saying that we don't process those types specially here (for other query languages such as MATCH and Gremlin, that's not the case) and telling them to go look at the integration tests for examples of passing in parameters.

Reply:
I meant to say that ff somebody tries to pass in a date(time) parameter for RedisGraph, you should raise an error if someone tries to insert a date(time) parameter into a cypher RedisGraph query.

Reply:
Yeah pointing to the integration tests makes sense to me.

Reply:
Addressed both. How does this new comment look?